ui/components/TableOfContents/index.js

```diff
@@ -0,0 +1,28 @@
+import { Link } from '../';
+
+export const TableOfContents = ({ content }) => {
+  // Anything h2 and below
+  const headingReg = new RegExp(/(#{2}\s)(.*)/gm);
```

Comment:
It's a bit of a weird edge case, but this will match `## ` in the middle of lines rather than only at the start of the line. We might want to either of:
a) modify the regex to only match the start of lines
b) use a proper markdown parser

Reply:
That's a deliberate decision based on the idea that the markdown might not come through perfectly. I think eventually if it proves to be an issue we could/should use a proper markdown parser (which though?).

Do you think it's okay to merge as is or does it need to go back to the drawing board?

Reply:
It's also not resistant to e.g. 
```
this kind of heading style
-------------------------
or this
=====
```
Though that's all valid markdown

Reply:
Updated to be more parse-y